WolvenKit.Common/Oodle/OodleNative.cs

```diff
@@ -82,7 +82,7 @@ public enum OodleLZ_Decode : int
             IntPtr inputBuffer,
             long inputSize,
             IntPtr outputBuffer,
```

Comment:
Can we have the dllimport point to the file next  to the exe or do we still need to put it next to ours? Would be better if it would work with the config path.